### ISSUE TITLE
test #27 fails on emacs27

### ISSUE THREAD

**Issue by KryDos** [ejmr/php-mode#426]
Hi, 
I'm in current Emacs master branch (`GNU Emacs 27.0.50`, Ubuntu) and when I run `make test` (in php-mode repo) I'm getting this error for `php-mode-test-issue-27`. 

<details>
  <summary>Click to see the log</summary>

```
  Test php-mode-test-issue-27 backtrace:                                         
  signal(wrong-type-argument (stringp nil))                                    
  c-syntactic-re-search-forward("[      \n[:?;{=*/%&|,<>!@+-]" nil t t)        
  c-forward-label()                                                            
  c-guess-basic-syntax()                                                       
  c-indent-region(1 500 nil)                                                   
  php-cautious-indent-region(1 500)                                            
  indent-region(1 500)                                                         
  (let ((inhibit-message t)) (indent-region (point-min) (point-max)))          
  (progn (insert-file-contents (expand-file-name "issue-27.php" php-mo         
  (unwind-protect (progn (insert-file-contents (expand-file-name "issu                                                                                         
  (save-current-buffer (set-buffer temp-buffer) (unwind-protect (progn         
  (let ((temp-buffer (generate-new-buffer " *temp*"))) (save-current-b         
  (lambda nil (let ((temp-buffer (generate-new-buffer " *temp*"))) (sa         
  ert--run-test-internal(#s(ert--test-execution-info :test #s(ert-test         
  ert-run-test(#s(ert-test :name php-mode-test-issue-27 :documentation         
  ert-run-or-rerun-test(#s(ert--stats :selector t :tests [#s(ert-test          
  ert-run-tests(t #f(compiled-function (event-type &rest event-args) #         
  ert-run-tests-batch(nil)                                                     
  ert-run-tests-batch-and-exit()                                               
  command-line-1(("-L" "." "-l" "php-mode-test.el" "-f" "ert-run-tests         
  command-line()                                                               
  normal-top-level()                                                           
Test php-mode-test-issue-27 condition:                                         
    (wrong-type-argument stringp nil)                                          
   FAILED  31/52  php-mode-test-issue-27
```
</details>

I was trying to find the reason of it but I didn't succeed due to my pure knowledge. As far as I understand the whole emacs27 branch fails the test but I'm not really sure. 

I would like to send pull request with pleasure if someone can help me to figure out why the test fails. 

P.S.
I have to mention that the issue tested with `php-mode-test-issue-27` is NOT happening actually. Indentation works perfectly.

**Comment by zonuexe** [comment]
@KryDos Thanks for reporting.

**Comment by zonuexe** [comment]
@KryDos This problem was solved in #453. Thanks!
